index.md

```diff
@@ -408,6 +390,23 @@ Then, to activate the environment you just created, run:
 conda activate AstronomicalData
 ```
 
+#### Install libraries in an existing Conda environment
```

Comment:
Can you add (not recommended) here too? Otherwise this looks great!

Reply:
Done!